packages/database/src/schema.prisma

```diff
@@ -525,7 +525,7 @@ model AppellantSubmission {
   appellantPhoneNumber           String?
 
   // number entries
-  siteAreaSquareMetres Int?
+  siteAreaSquareMetres Float?
```

Comment:
Hi @Si3rr4wow, I am putting together the JSON schema for the appeal data. In that format, `number` covers any numeric value (INT, FLOAT, DECIMAL, LONG), so currently I'm using `"multipleOf": 0.1` to indicate a single decimal point...

Do you know how the front office validates this value? I can align my side, or maybe a `Decimal?` type would work better on SQL?

Reply:
Hey Fabio, I was told we were making this change to align with you 😅 We still want this to be an integer on the front end

Reply:
Are you saying we can just leave this an an int and BO won't have a problem with that?

Reply:
No I meant `Float -> Decimal` :-)
Let me check with @beejjacobs ...

Reply:
Oh sorry, done :)